import-export-format.md

```diff
@@ -0,0 +1,183 @@
+Import / Export Format
+======================
+
+Export Format
+-------------
+When run in `export` mode, the import/export utility creates a directory tree representing the designated resource node and its child nodes:
```

Comment:
Please change: `designated resource node` 
..to `designated resource`
- and `child nodes` 
..to `child resources (where 'b' is a child of 'a' in this case, /a/b)`